src/tools/perf/perftest_params.c

```diff
@@ -312,7 +301,28 @@ static ucs_status_t verify_daemon_params(ucx_perf_params_t *params)
         return UCS_ERR_UNSUPPORTED;
     }
 
-    params->ucp.is_daemon_mode = 1;
+    return UCS_OK;
+}
+
+static ucs_status_t init_daemon_params(ucx_perf_params_t *params)
+{
+    struct sockaddr_storage *local_addr  = &params->ucp.dmn_local_addr;
+    struct sockaddr_storage *remote_addr = &params->ucp.dmn_remote_addr;
+
+    /* Return if both daemon addresses are not configured */
+    if ((0 == local_addr->ss_family) && (0 == remote_addr->ss_family)) {
+        return UCS_OK;
+    }
```

Comment:
shouldn't this be in check_daemon_params?

Reply:
can we document this distinction in the names of the functions and/or in some comment before thr functions themselves?

Reply:
sure